src/main/java/com/epam/university/java/core/task055/Task055.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.university.java.core.task055;
+
+import java.util.Collection;
+
+/**
+ * Read XML.
+ * Implement a programme that get a data about Saint Petersburg residential buildings from
+ * XML file with buildings passports.
+ * Table comes from: https://data.gov.spb.ru/opendata/7840013199-passports_houses/.
+ */
+public interface Task055 {
+
+    /**
+     * Set filepath to read.
+     * @param path - full path to file to read.
+     */
+    void setInputFile(String path);
```

Comment:
This is not a really good approach when you provide your component with a global state, it's better having a method like:

```java
public interface Task055 {
    ProcessingContext createContext(String filepath);
}

public class ProcessingContext {
    Collection<HouseDefinition> oldestHouses();

    // rest of methods from this class
}
```

In this case your class will create a separate context for every file and this context will provide all the necessary methods to perform actions for the given file.